code/modules/antagonists/heretic/knowledge/rust_lore.dm

```diff
@@ -66,6 +67,8 @@
 /datum/heretic_knowledge/rust_fist/on_gain(mob/user)
 	RegisterSignal(user, COMSIG_HERETIC_MANSUS_GRASP_ATTACK, .proc/on_mansus_grasp)
 	RegisterSignal(user, COMSIG_HERETIC_MANSUS_GRASP_ATTACK_SECONDARY, .proc/on_secondary_mansus_grasp)
+	var/datum/antagonist/heretic/our_heretic = IS_HERETIC(user)
+	our_heretic?.increase_rust_strength(1)
```

Comment:
`our_heretic` should never ever be `null`, so you don't need the `?.` here
```suggestion
	our_heretic.increase_rust_strength(1)
```
(even if it was null for some reason, we'd want it to error because something broke)